Source/WebCore/platform/graphics/SourceBufferPrivate.cpp

```diff
@@ -564,10 +570,10 @@ void SourceBufferPrivate::updateTrackIds(Vector<std::pair<AtomString, AtomString
     }
 }
 
-void SourceBufferPrivate::setClient(SourceBufferPrivateClient& client)
+void SourceBufferPrivate::setClient(Ref<SourceBufferPrivateClient> client)
```

Comment:
Why this change, this will trigger count churning for no good reason.

Reply:
this is an unfortunate copy of another WIP when SourceBufferPrivate is called from a different thread, and so you need to pass a Ref. I'll remove it